railties/test/generators/plugin_generator_test.rb

```diff
@@ -830,4 +813,15 @@ def assert_match_sqlite3(contents)
         assert_match(/group :development do\n  gem 'sqlite3'\nend/, contents)
       end
     end
+
+    def simulate_application_existence
```

Comment:
I think the typical nomenclature might be more like

```suggestion
    def with_simulated_app
```

Reply:
Thanks, @jonathanhefner – fair point. Force-pushing a single commit that merges my previous commits plus this change.